### ISSUE TITLE
AAPT warnings because of wrong syntax in ts_ratingbar_full.xml

### ISSUE THREAD

**Issue by herbeth1u** [trustedshops/trustedshops-android-sdk#10]
**SDK version:** 1.9.4

Hello.
When compiling the SDK, AAPT throws these warnings:
```
Android resource linking failed
ts_ratingbar_full.xml:3: AAPT: warn: generated id 'android:id/background' for external package 'android'.
ts_ratingbar_full.xml:7: AAPT: warn: generated id 'android:id/progress' for external package 'android'.
ts_ratingbar_full.xml:5: AAPT: warn: generated id 'android:id/secondaryProgress' for external package 'android'.
```

Indeed, in `ts_ratingbar_full.xml`, ids are referenced with `@+android:id/background`, `@+android:id/secondaryProgress` and `@+android:id/progress` instead of `@android:id/background`, `@android:id/secondaryProgress` and `@android:id/progress` (references to resources shouldn't use "+").

Would it be possible to fix these warnings?
Thank you.

Edit: Edited the issue; these are only warnings although AAPT's output may let you think otherwise.

**Comment by superus8r** [comment]
This is weird... It seems like something was outdated in the SDK.
I believe this should have been fixed now, see PR #11 (thanks to @herbeth1u for reporting this).

**Comment by mrbipbip** [comment]
Hi, sorry to write in closed issue but this fix doesn't seems to be part of any released version (last 1.9.4 version if from 2018). Could you please release a new version with this fix included?
Bests.
